.papr.yml

```diff
@@ -70,6 +70,8 @@ inherit: true
 context: f25-experimental-api
 env:
   CONFIGOPTS: '--enable-experimental-api'
+  CFLAGS: '-Werror=unused-variable'
```

Comment:
Sure, that sounds great to me! I rebased and turned it on directly in the build scripts. ⬇️ Let's see what happens.